spec/models/book_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe Book do
+  it 'is valid with valid attributes' do
+    book = build(:book)
+
+    expect(book).to be_valid
+  end
+
+  it 'is not valid without a genre' do
```

Comment:
We can validate the presence of attributes in one line such as the following:
`is_expected.to validate_presence_of(:genre)`.

See: https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax